cmd/main.go

```diff
@@ -209,3 +218,21 @@ func triggerInterval(intervalChannel chan bool, productionInterval time.Duration
 	time.Sleep(productionInterval)
 	intervalChannel <- true
 }
+
+func addPeertoDiscoveryList(ip, port string) {
+	client := http.Client{}
+	//send initally and then wait every 5 minutes
+	r, err := requests.AddPeerToDiscoveryRequest(ip, port)
+	if err != nil {
+		log.Fatalf("Failed to make Add Peer request: %s", err.Error())
```

Comment:
Let's not fatally exit if it fails to make the request. It should still be able to function.

Reply:
I'll change to to a normal log, and continually try to send.